docker-compose.yml

```diff
@@ -0,0 +1,58 @@
+version: '3.4'
+
+services:
+  workspace:
+    build: .
+    # or use the latest docker image
+    # image: kbase/workspace_deluxe:latest
```

Comment:
Our images are stored on GHA so I don't this would actually work would it?

Reply:
I've added some instructions to the README on how to pull the latest docker images from the GitHub container repo.

Reply:
:+1: